ScoutSuite/output/data/html/partials/azure/services.securitycenter.pricings.html

```diff
@@ -0,0 +1,25 @@
+<!-- Storage Accounts partial -->
+<script id="services.securitycenter.pricings.partial" type="text/x-handlebars-template">
+    <div class="list-group-item active">
+        <h4 class="list-group-item-heading">{{name}}</h4>
+    </div>
+
+    <div class="list-group-item">
+        <h4 class="list-group-item-heading">Information</h4>
+        <div class="list-group-item-text item-margin">Pricing Name: <span id="securitycenter.pricings.{{@key}}.name">{{name}}</span></div>
+        <div class="list-group-item-text item-margin">Pricing Tier: <span id="securitycenter.pricings.{{@key}}.pricing_tier">{{ pricing_tier }}</span></div>
+    </div>
+</script>
+
+<script>
+    Handlebars.registerPartial("services.securitycenter.pricings", $("#services\\.securitycenter\\.pricings\\.partial").html());
+</script>
+
+<!-- Single Pricings template -->
+<script id="single_securitycenter_pricing-template" type="text/x-handlebars-template">
+    <div style="text-align: right; padding-right: 10px; text-weight: bold;"><a href="javascript:hidePopup()">X</a></div>
+    {{> services.securitycenter.pricings}}
```

Comment:
Please use the modal template instead. See this [PR](https://github.com/nccgroup/ScoutSuite/pull/132/files#diff-fa8c67fc19bd13c3d67c401c70236778) as an example.

Reply:
Nice to see you guys are standardizing practices. Might be worth highlighting all of these norms in the wiki?

Reply:
^^^